cmd/tailscale/cli/status.go

```diff
@@ -146,6 +146,9 @@ func runStatus(ctx context.Context, args []string) error {
 
 	var buf bytes.Buffer
 	f := func(format string, a ...any) { fmt.Fprintf(&buf, format, a...) }
+
+	f("%-15s %-20s\n", "tailnet name", st.CurrentTailnet.MagicDNSSuffix)
```

Comment:
You should guard against CurrentTailnet being nil.

I wonder if the output format should be more like a header at the top, like Health check above is, rather than trying to blend in with the peers.

Reply:
Thanks! Maybe the peers should have a heading as well, but I don't know if the heading should be "Peers", or "Machines" as in admin console, or  "Nodes" as in `tailscale status -web`.